src/components/ExpandableBox/ExpandableBox.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react'
+import { spring, Motion } from 'react-motion'
+import {
+  styled,
+  theme,
+  spring as springConf,
+  unselectable,
+  Text,
+} from '@aragon/ui'
+import arrow from './assets/arrow.svg'
+
+class ExpandableBox extends React.Component {
+  state = {
+    selfExpanded: false,
+    drawerHeight: 0,
+  }
+  componentDidMount() {
+    this.setState({ drawerHeight: this.getDrawerHeight() })
+  }
+  getDrawerHeight() {
+    const rect = this.drawerElt.getBoundingClientRect()
+    return rect.height
```

Comment:
Yeah… I was trying to avoid it (by resetting the height each time a click occur), but it still can break in some cases. I’m not sure we need the complexity of react-resize-detector for this case, a throttled `document.onResize` should be enough.